driver/test/java/io/crate/client/jdbc/integrationtests/MetaDataITest.java

```diff
@@ -37,21 +43,21 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
 
-public class CrateJDBCMetaDataIntegrationTest extends CrateJDBCIntegrationTest {
+public class MetaDataITest extends BaseIntegrationTest {
 
-    private static Connection connection;
+    private static Connection CONN;
```

Comment:
Should this be non-static ? `setUpTest` is run per test, so it's re-assigned per test anyways?

Reply:
good catch